recipe/build.sh

```diff
@@ -0,0 +1,30 @@
+export PREFIX_NSIS=$PREFIX/NSIS
+
+pushd binary
+cp -r . "$PREFIX_NSIS"
+rm -rf $PREFIX_NSIS/Docs
+rm -rf $PREFIX_NSIS/Examples
+popd
+
+pushd plugins
+cp "UAC/U/UAC.dll" "$PREFIX_NSIS/Plugins/x86-unicode/"
+cp "untgz/Plugins/x86-unicode/untgz.dll" "$PREFIX_NSIS/Plugins/x86-unicode/"
+cp "UnicodePathTest/Plugin/UnicodePathTest.dll" "$PREFIX_NSIS/Plugins/x86-unicode/"
+popd
+
+cd src
+scons \
+  CC="${CC}" CXX="${CXX}" APPEND_CCFLAGS="${CXXFLAGS}" APPEND_LINKFLAGS="${LDFLAGS}" \
```

Comment:
Re `APPEND_CCFLAGS="${CXXFLAGS}"`: yes, this means `CXXFLAGS` will also get used for the C compilations, so not perfect either. But I found it a bit clearer than adding `$BUILD_PREFIX/bin/gcc`.